src/curated-corpus/pages/CuratedCorpusLandingPage/CuratedCorpusLandingPage.tsx

```diff
@@ -47,66 +45,64 @@ export const CuratedCorpusLandingPage = (): JSX.Element => {
 
   return (
     <ApolloProvider client={client}>
-      <MuiPickersUtilsProvider utils={LuxonUtils}>
-        <HeaderConnector
-          productName="Curated Corpus"
```

Comment:
Removing the `MuiPickersUtilsProvider` from this page breaks the Prospecting page which seems to be the only place where individual components/forms were not being wrapped in it already. Given it's under active transition to MUI 5, too, I chose not to add it to the Prospecting page code as it won't be needed with the new MUI X pickers.

CC @Herraj